app/assets/javascripts/fae/form/_filtering.js

```diff
@@ -31,17 +31,17 @@ Fae.form.filtering = {
     // hardcode _this because this === Fryr object
     var _this = Fae.form.filtering;
     var post_url = _this.$filter_form.attr('action');
-    var $results_table = ($(".js-results-table").length) ? $(".js-results-table").first() : _this.$filter_form.find('table');
- 
+    var $results_table = ($(".js-results-table").length) ? $(".js-results-table").first() : _this.$filter_form.next('table');
```

Comment:
I can't speak to that. From my understanding, the PR that introduced this code was heavily vetted, so I ASSume it should be fine...